server/mailer/index.js

```diff
@@ -68,4 +68,38 @@ const sendSignUpConfirmationLink = (req, resp) => {
     .catch(() => resp.sendStatus(400));
 };
 
-module.exports = { sendInvitation, sendSignUpConfirmationLink };
+const sendResetPasswordLink = (req, resp) => {
+  const { email: receiver, displayName, resetToken } = resp.locales;
+  const { protocol } = req;
+  const host = req.get('host');
+  const url = `${protocol}://${host}`;
```

Comment:
do you need `protocol`? Isn't `host` is enough?

Reply:
It works without protocol, but I have added this because on production server we have https
and I thought this solution is more bulletproof. 
I can remove, what do you think?

Reply:
it will be redirected automatically - you don't need to use `protocol`